fill/actuators/BUILD

```diff
@@ -19,9 +19,15 @@ cc_library(
 
 cc_library(
     name = "ball_valve",
-    srcs = ["ball_valve.cc"],
     hdrs = ["ball_valve.h"],
-    deps = [":actuator"],
+    srcs  = ["ball_valve.cc"], 
+    deps = ["//fill/lib/MockWiringPi:mock_wiringpi"],
+    includes = ["."],
+    # copts = select({
```

Comment:
Please remove the commented lines here. And in the stepper motor driver's BUILD too if you don't mind